tmt/export.py

```diff
@@ -157,7 +162,7 @@ def export_to_nitrate(test, create, general):
         'purpose-file': test.description,
         'hardware': test.node.get('extra-hardware'),
         'pepa': test.node.get('extra-pepa'),
-        }
+    }
```

Comment:
Please, keep the ending `}` aligned with the previous line. Same above.

Reply:
fixed